frontend/common/src/components/form/Input/Input.tsx

```diff
@@ -87,6 +87,7 @@ const Input: React.FunctionComponent<InputProps> = ({
           readOnly={readOnly}
           aria-required={rules.required ? "true" : undefined}
           aria-invalid={error ? "true" : "false"}
+          aria-describedby={error ? `${id}-error` : undefined}
```

Comment:
Nice, I see the relationship for the described by in my accessibility tools!
![image](https://user-images.githubusercontent.com/8978655/203420417-ad3f0bc9-5a37-4585-a82c-a3e4c41f7acc.png)